plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -479,6 +539,7 @@ def main():
         state=dict(type='str', default="present", choices=["absent", "present", "blocked", "unblocked"]),
         username=dict(type='str', required=True),
         password=dict(type='str', no_log=True),
+        reset_password=dict(type='bool', default=False, no_log=True),
```

Comment:
This boolean value is **not** a secret.
```suggestion
        reset_password=dict(type='bool', default=False, no_log=False),
```